x-pack/plugins/enterprise_search/public/applications/workplace_search/views/groups/components/group_users_table.tsx

```diff
@@ -58,10 +53,10 @@ export const GroupUsersTable: React.FC = () => {
   return (
     <>
       <EuiTable>
-        <TableHeader extraCell={isFederatedAuth} headerItems={headerItems} />
+        <TableHeader extraCell={false} headerItems={headerItems} />
```

Comment:
I'd just leave `extraCell` off entirely since it looks like it doesn't default tot rue
```suggestion
        <TableHeader headerItems={headerItems} />
```